src/RandomEnchants.cpp

```diff
@@ -39,16 +39,6 @@ class REConfig : public WorldScript
     // Load Configuration Settings
     void SetInitialWorldSettings()
     {
-        RandomEnchantEnabled = sConfigMgr->GetBoolDefault("RandomEnchants.Enabled", true);
-        RandomEnchantAnnounce = sConfigMgr->GetBoolDefault("RandomEnchants.Announce", true);
-        Crafted = sConfigMgr->GetBoolDefault("RandomEnchants.OnCreate", 1);
-        Looted = sConfigMgr->GetBoolDefault("RandomEnchants.OnLoot", 1);
-        QuestReward = sConfigMgr->GetBoolDefault("RandomEnchants.OnQuestReward", 1);
-        HighQuality = sConfigMgr->GetIntDefault("RandomEnchants.HighQuality", 1);
-        LowQuality = sConfigMgr->GetIntDefault("RandomEnchants.LowQuality", 1);
-        Chance1 = sConfigMgr->GetIntDefault("RandomEnchants.Chance1", 60);
-        Chance2 = sConfigMgr->GetIntDefault("RandomEnchants.Chance2", 65);
-        Chance3 = sConfigMgr->GetIntDefault("RandomEnchants.Chance3", 70);
```

Comment:
I don't see this variables getting the values from the config in any other place than this, you sure you are supposed to delete this?

Reply:
Absolutely not sure at all but @BarbzYHOOL told me to , who am I argue .

Reply:
Actually I’m done here , they way you treat my freinds is bad , do your own work from now on I’ll only be doing it privately

Reply:
Whe y’all decide to stop being bullies to Poszer and talmortis and I might decide to contribute here again